dev/run_docker_compose.sh

```diff
@@ -25,17 +25,17 @@ ARROW_SRC=$(realpath "${PWD}/..")
 : ${DOCKER_COMPOSE_CONF:="${ARROW_SRC}/docker-compose.yml"}
 
 docker_compose() {
-  ${DOCKER_COMPOSE} -f ${DOCKER_COMPOSE_CONF} $@
+  ${DOCKER_COMPOSE} -f ${DOCKER_COMPOSE_CONF} "$@"
```

Comment:
I think We should get rid of this script, I don't see any advantages it adds to plain docker-compose commands.
Docker-compose automatically searches for `docker-compose.yml` upwards in the folder hierarchy, thus it finds `arrow/docker-compose.yml` properly.

Reply:
Could you open a JIRA issue for this?
I'm not sure whether this is useful for us or not.